src/modules/color/index.ts

```diff
@@ -246,14 +246,16 @@ export class ColorModule {
    * @param options.includeAlpha Adds an alpha value to the color (RGBA). Defaults to `false`.
    *
    * @example
-   * faker.color.rgb() // '#8be4ab'
+   * faker.color.rgb() // '#0d7f26'
    * faker.color.rgb({ prefix: '0x' }) // '0x9ddc8b'
    * faker.color.rgb({ casing: 'upper' }) // '#B8A51E'
    * faker.color.rgb({ casing: 'lower' }) // '#b12f8b'
    * faker.color.rgb({ prefix: '#', casing: 'lower' }) // '#eb0c16'
    * faker.color.rgb({ format: 'hex', casing: 'lower' }) // '#bb9d17'
    * faker.color.rgb({ format: 'css' }) // 'rgb(216, 17, 192)'
    * faker.color.rgb({ format: 'binary' }) // '00110010 00001000 01110110'
+   * faker.color.rgb({ includeAlpha: true }) // '#f96efb80'
```

Comment:
This is just a slight suggestion, but we could use another value for the alpha, so it is not exactly half in the middle / grey

```suggestion
   * faker.color.rgb({ includeAlpha: true }) // '#f96efb5e'
```